test/index.spec.js

```diff
@@ -684,12 +684,27 @@ describe('onBuild()', () => {
     })
   })
 
-  test('generates an ipx function by default', async () => {
+  // Skipped while edge images are off by default
+  test.skip('generates an edge ipx function by default', async () => {
```

Comment:
Was this test being skipped for debugging purposes?

Reply:
Ah yep, I had added that skip and the comment above as I thought it was for edge functions, but it's a regular function. Unskipped & comment removed :)